### PR TITLE
feat(provider/kubernetes): V2 Annotations

### PR DESCRIPTION
**Pull request by lwander**

### CODE REVIEW

File: clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/v2/description/KubernetesManifestSpinnakerRelationships.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+ *
+ */
+
+package com.netflix.spinnaker.clouddriver.kubernetes.v2.description;
+
+import lombok.Data;
+
+import java.util.List;
+
+@Data
```
Comment (duftler) [inline clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/v2/description/KubernetesManifestSpinnakerRelationships.java]: Could use `@Builder` here.

Comment (lwander) [reply clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/v2/description/KubernetesManifestSpinnakerRelationships.java]: SGTM, `@Data` will still be needed for serializing/deserializing later

**Comment by lwander** [comment]
spinnaker/spinnaker#1902

**Comment by lwander** [comment]
spinnaker/spinnaker#1922
spinnaker/spinnaker#1916

**Pull request merged**